homework5/server/models/answers.js

```diff
@@ -0,0 +1,34 @@
+const Answer = require("./answer");
+
+class Answers {
+  constructor() {
+    this.answers = [];
+  }
+
+  newAnswer(answer) {
+    if (this.answerAlreadyExist(answer)) {
+      for (var i = 0; i < this.allAnswers; i++)
```

Comment:
Incearca sa folosesti `const` sau `let` in locul lui `var`.

Reply:
De asemenea poti sa folosesti `forEach` sau daca nu vrei sa ai un array pentru occurrence poti sa imiti un Map care va face answers[answer] si le vei contoriza direct. Daca nu exista iti va crea unul nou.